src/events/audit/guildAuditAnnouncementSend.ts

```diff
@@ -22,15 +23,14 @@ export default class extends AuditEvent {
 					message_source_id: message.id,
 					message_result_id: resultMessage.id
 				},
-				tags: {
-					shard: (this.client.options.shards as number[])[0].toString(),
-					user_id: message.author.id,
-					guild_id: message.guild?.id!,
-					channel_id: channel.id,
-					action: AuditAnnouncementAction.Send
-				}
-			}
-		]);
+				tags: this.formTags({
+					[Tags.Shard]: (this.client.options.shards as number[])[0].toString(),
+					[Tags.User]: message.author.id,
+					[Tags.Guild]: message.guild?.id!,
```

Comment:
guild is never null in this event.
```suggestion
					[Tags.Guild]: message.guild!.id,
```